### ISSUE TITLE
Packaging?

### ISSUE THREAD

**Issue by PythonCHB** [UWPCE-PythonCert/PythonCertDevel220#6]
I used to have a packaging module that I think got lost in the shuffle in the last re-do.

I have the materials here:

https://uwpce-pythoncert.github.io/PythonCertDevel/modules/Packaging.html

source here:

https://github.com/UWPCE-PythonCert/PythonCertDevel/blob/master/source/modules/Packaging.rst

I think it's actually pretty critical for students to be able to make a package of their code. Once you get more than one file for a project, it's really best to package it up (including tests).

Also, it's a good time to cover all the options for importing, which is pretty confusing to newbies.

Squeeze it in before the advanced testing topic?

**Comment by pythonsavvy** [comment]
Yes I'll squeeze it in. This actually resolves one of our to dos. I have found the students are totally confused about modules, packages and how import works. I will add this to the preclass materials. Thanks Chris
